lnwallet/btcwallet/psbt.go

```diff
@@ -49,7 +50,8 @@ var (
 //
 // This is a part of the WalletController interface.
 func (b *BtcWallet) FundPsbt(packet *psbt.Packet, minConfs int32,
-	feeRate chainfee.SatPerKWeight, accountName string) (int32, error) {
+	feeRate chainfee.SatPerKWeight, accountName string,
+	changeScope *waddrmgr.KeyScope) (int32, error) {
 
```

Comment:
I just came to my eyes that in line 67 in the comment, we could also add the P2TR type for the default account?

Reply:
good point I think